graphql-kotlin-spring-server/src/main/kotlin/com/expediagroup/graphql/spring/exception/SimpleKotlinGraphQLError.kt

```diff
@@ -32,13 +32,12 @@ open class SimpleKotlinGraphQLError(
 ) : GraphQLError {
     override fun getErrorType(): ErrorClassification = errorType
 
-    override fun getExtensions(): Map<String, Any> {
-        val newExtensions = mutableMapOf<String, Any>()
+    override fun getExtensions(): Map<String, Any> =
         if (exception is GraphQLError && exception.extensions != null) {
-            newExtensions.putAll(exception.extensions)
+            exception.extensions
```

Comment:
i believe the intent here was to ensure it is mutable map so we could add to it afterwards if needed

Reply:
actually nvm this should be fine. if we need to add to the extensions then we can modify it to mutable map at that time